SECURITY.md

```diff
@@ -0,0 +1,59 @@
+# Security Policy
+
+## Reporting a Vulnerability
```

Comment:
I would up front credit where this policy came from (containerd) - see how Rook did this at the very front, crediting Envoy and CoreDNS: https://github.com/rook/rook/blob/master/SECURITY.md#security-release-process